### ISSUE TITLE
Second level nested property values ignore adminbro settings

### ISSUE THREAD

**Issue by dmitryLagoda** [SoftwareBrothers/admin-bro#220]
```
const schema1 = new Schema({
  name: String,
  task: new Schema({
    name: String,
    subtask: new Schema({
      name: String,
      subtask1: String
    }, { _id: false })
  }, { _id: false })
});
const TestModel = mongoose.model('TestModel', schema1);

const adminBro = new AdminBro({
  resources: [
    {resource: TestModel, options: {
      properties: {
        'task.name' : {
          type: 'datetime'
        },
        'task.subtask.name' : {
          type: 'datetime'
        },
      }
    }},
  ],
})
```
We have 2 levels of nesting for **TestModel**
1) First level nested  **TestModel.task.name** works fine, has type ='datetime'. 
2) Second level nested **TestModel.task.subtask.name** doesn't work, the type is not "datetime"
Repeats for all types of settings: `label, components, type etc.`
<img width="739" alt="Screen Shot 2019-12-10 at 11 48 01 PM" src="https://user-images.githubusercontent.com/2717056/70572840-4a686500-1ba9-11ea-8d8c-6805f633a1ce.png">

**Comment by wojtek-krysiak** [comment]
crazy issue - on it....

**Comment by wojtek-krysiak** [comment]
fixed in admin-bro@1.5.2

**Comment by dmitryLagoda** [comment]
Thank you!
